onnxruntime/core/providers/nnapi/nnapi_builtin/builders/helper.cc

```diff
@@ -232,8 +234,10 @@ bool HasValidQuantizationZeroPoints(const InitializedTensorSet& initializers, co
 
         std::unique_ptr<uint8_t[]> unpacked_tensor;
         size_t tensor_byte_size;
-        auto status = onnxruntime::utils::UnpackInitializerData(zero_tensor, node.ModelPath(),
-                                                                unpacked_tensor, tensor_byte_size);
+        auto status = onnxruntime::utils::UnpackInitializerData(
+            zero_tensor,
+            node.ModelPath(),  // We don't support external initializers in NNAPI
+            unpacked_tensor, tensor_byte_size);
```

Comment:
Is this comment relevant? The unpack will turn something from an external initializer into in-memory data won't it?

Reply:
Put at the wrong position